packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/testable/ServiceTestDataState.ts

```diff
@@ -366,7 +366,10 @@ export class ServiceTestDataState {
       this.testSuiteState.testableState.serviceEditorState.service;
     const execution = service.execution;
     let runtimes: Runtime[] = [];
-    if (execution instanceof PureSingleExecution) {
+    if (
+      execution instanceof PureSingleExecution &&
+      execution.runtime !== undefined
```

Comment:
&& execution.runtime

Reply:
tried the suggestion out was giving a lint error, didnt dvelve much into why